packages/react-icons/src/utils/createSvgIcon.ts

```diff
@@ -21,7 +21,7 @@ const createSvgIcon = <TProps = {}>({ svg, displayName }: SvgIconCreateFnParams<
       {
         ...containerProps,
         ...nativeProps,
-        className: css(classes.root, className),
+        className,
```

Comment:
this doesn't seem right. 

classes.root is no longer applied to the container. We should either remove the span if the concern is an extra element, and push root to the svg, or, keep pushing classes.root to the span.

Reply:
there were no styles defined on root as i removed it: https://github.com/microsoft/fluentui/blob/4e058b89af037da8b421ecc16b628238de011107/packages/react-icons/src/utils/SvgIcon.styles.ts#L11

Reply:
We will need to add styles on the root, we discussed with @dzearing about adding the following styles:
```
speak: none;
vertical-align: middle;
display: inline-block;
```

I just didn't manage to create the PR yet. So I would suggest keeping the `classes.root` there

Reply:
Added root back and also added the css fixes